README.md

```diff
@@ -1,4 +1,4 @@
-# ![nf-core/metapep](docs/images/nf-core-metapep_logo_light.png#gh-light-mode-only) ![nf-core/metapep](docs/images/nf-core-metapep_logo_dark.png#gh-dark-mode-only)
+# ![nf-core/metapep](docs/images/nf-core/metapep_logo_light.png#gh-light-mode-only) ![nf-core/metapep](docs/images/nf-core/metapep_logo_dark.png#gh-dark-mode-only)
```

Comment:
That broke the logo, had that with all pipeline updates, I guess thats a template bug.
```suggestion
# ![nf-core/metapep](docs/images/nf-core-metapep_logo_light.png#gh-light-mode-only) ![nf-core/metapep](docs/images/nf-core-metapep_logo_dark.png#gh-dark-mode-only)
```